<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>188</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="start_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>Launch Application</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="otziv_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(148, 245, 255);</string>
      </property>
      <property name="text">
       <string>Leave feedback for the app</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 33, 33);</string>
      </property>
      <property name="text">
       <string>Quit From Application</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-110</x>
     <y>340</y>
     <width>1171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>368</y>
     <width>160</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="review_group_btn_2">
    <item>
     <widget class="QLabel" name="ask_line">
      <property name="text">
       <string>Напишите свой отзыв здесь</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="Review_Text_Edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 0);</string>
      </property>
      <property name="text">
       <string>Сохранить отзыв</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-260</x>
     <y>158</y>
     <width>1491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>761</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Ultimate Editor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
